astropy/visualization/tests/test_time.py

```diff
@@ -8,7 +8,10 @@
 from astropy.time import Time
 from astropy.visualization.time import time_support
 
-plt = pytest.importorskip("matplotlib.pyplot")
+try:
+    from matplotlib.figure import Figure
```

Comment:
Should we just `pytest.importorskip("matplotlib")` and be done with it, instead of try-except?

Reply:
`importorskip` doesn't work at the module level. I tried to keep the diff minimal here.
Though, it would still look better to use `HAS_MATPLOTLIB` here, so I'll do just that